etl/models/models.go

```diff
@@ -59,6 +72,45 @@ type JetDrop struct {
 	RecordAmount   int
 }
 
+type State struct {
+	RecordRef    []byte `gorm:"primary_key;auto_increment:false"` // State reference.
+	Type         StatesType
+	RequestRef   []byte // Reference to the corresponding request.
+	ParentRef    []byte // Reference to the parent object that caused creation of the given object.
+	ObjectRef    []byte
+	PrevStateRef []byte // Reference to a previous state.
```

Comment:
would you want to use models.Reference instead of []byte for fields such as RecordRef, RequestRef, ObjectRef and e.t.c.?